composer.json

```diff
@@ -27,7 +27,8 @@
         "oat-sa/extension-tao-delivery-rdf": ">=14.0.0",
         "oat-sa/extension-tao-testtaker": ">=8.0.0",
         "oat-sa/extension-tao-proctoring": ">=20.0.0",
-        "oat-sa/extension-tao-clientdiag": ">=8.0.0"
+        "oat-sa/extension-tao-clientdiag": ">=8.0.0",
+        "oat-sa/extension-tao-group": "~7.7"
```

Comment:
If understand right, `~` would deny the next breaking release, therefore everything `>=8.0`. Lets keep the current convention and require with `>=`.
On other note, I don't think there is something specific in version `7.7` that is necessary for this extension, so maybe lets have the same requirement as [here](https://github.com/oat-sa/extension-tao-delivery-rdf/blob/master/composer.json#L27), and require `>=7.0.0`?